Documentation/gettingstarted/encryption.rst

```diff
@@ -224,26 +275,62 @@ Troubleshooting
 
    * Routing rules matching on fwmark:
 
-     .. code:: bash
+     .. code-block:: shell-session
 
-        ip rule list
+        $ ip rule list
         1:	from all fwmark 0xd00/0xf00 lookup 200
         1:	from all fwmark 0xe00/0xf00 lookup 200
         [...]
 
-   * Content of routing table 200
+   * Content of routing tables:
+
+    .. tabs::
+
+        .. group-tab:: GKE
+
+            Content of routing table 200:
+
+             .. code-block:: shell-session
+
+                $ ip route list table 200
+                local 10.60.0.0/24 dev cilium_vxlan proto 50 scope host
+                10.60.1.0/24 via 10.60.0.1 dev cilium_host
+
+        .. group-tab:: IPAM ENI mode
+
+            Content of routing table 200:
+
+            .. code-block:: shell-session
+
+               $ ip route list table 200
+               10.93.0.0/16 dev cilium_host mtu 9001
+               local 10.135.0.0/16 dev eth0 proto 50 scope host
 
-     .. code:: bash
+            Content of per-ENI routing table:
 
-        ip route list table 200
-        local 10.60.0.0/24 dev cilium_vxlan proto 50 scope host
-        10.60.1.0/24 via 10.60.0.1 dev cilium_host
+                Check if routing rules exist for the the IP address of ``cilium_host`` interface.
+
+                .. note::
+                    Skip this step if ``endpoint-routes`` is enabled.
+
```

Comment:
`endpoint-routes` is always enabled in IPAM ENI mode, so the step can always be ignored.

I don't think we need the note at all here?